.github/workflows/verifications.yml

```diff
@@ -33,8 +33,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # The .x indicates "the most recent one"
-        node: [19.x, 18.x, 17.x, 16.x, 14.x, 14.17.0, 12.x, 12.22.0]
+        node: [12.22.0, 12, 14.17.0, 13, 16, 18, 20, 22]
```

Comment:
Could you keep all previous node versions? I'll remove the outdated ones in #616

Reply:
```suggestion
        node: [12.22.0, 12, 14.17.0, 14, 16, 17, 18, 19, 20, 22]
```